test/teleport_router.ts

```diff
@@ -1,33 +1,16 @@
 import { Account } from '@shardlabs/starknet-hardhat-plugin/dist/src/account';
```

Comment:
In general I miss original test declarations. Keeping them will make alignment easier.

Reply:
Another general remark: indentation of solidity code needs to be fixed.